CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+Contributing to the Node.js Hardware Working Group
+==================================================
+
+## Code of Conduct
+
+### Purpose
+
+The core of the NodeBots community is the people in it. Our primary goal is ensure this community is as inclusive to the largest number of people, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, gender identity and expression, sexual orientation, ability, physical appearance, body size, race, age, socioeconomic status, religion (or lack thereof), or other marginalized aspect of attendees.
```

Comment:
I'd love to hear everyone's thoughts on this, I went back and forth myself (this wasn't an oversight).

I feel like this working group exists to further the technical goals of those working within the NodeBots community, and NodeBots is indeed known for its community, so this is a good place to namedrop it. That said, I do question the validity of my assumption that the NodeBots community encompasses everything that is Node.js-hardware related.

Thoughts?